src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2091,19 +2091,19 @@ private void RemoveWorseMembers<TMember>(ArrayBuilder<MemberResolutionResult<TMe
         private static ParameterSymbol GetParameterOrExtensionParameter<TMember>(int argIndex, MemberAnalysisResult result, ImmutableArray<ParameterSymbol> parameters, TMember member)
```

Comment:
>GetParameterOrExtensionParameter [](http://example.com/codeflow?start=39&length=32)

Let's make this a local function in BetterFunctionMember. It looks like it is used only in that function and it is a source of mismatch between parameters and arguments,